src/core/lib/security/credentials/fake/fake_credentials.h

```diff
@@ -75,6 +77,10 @@ class grpc_md_only_test_credentials : public grpc_call_credentials {
   void cancel_get_request_metadata(grpc_credentials_mdelem_array* md_array,
                                    grpc_error* error) override;
 
+  std::string debug_string() const override {
```

Comment:
nit: I find it a little annoying to see `std::string` internally and `grpc::string` in public header. I think it might be better if you just use `grpc::string` and thus you do not need to add the include everywhere.